site-scrapers/index.js

```diff
@@ -2,24 +2,23 @@ const fs = require("fs");
 
 let scrapers = [];
 
-const ls = fs
-    .readdirSync("./site-scrapers", { withFileTypes: true })
-    .filter((item) => !item.isDirectory())
-    .filter((item) => item.name !== "index.js")
-    .map((item) => item.name);
-
-ls.map((fileName) => {
-    scraper = require(`./${fileName}`);
-    scrapers.push(scraper);
-});
-
 if (process.argv.length > 2) {
-    scrapers = []; // args override directory list allowing single site runs
     let scraper;
     for (let i = 2; i < process.argv.length; i++) {
         scraper = require(`./${process.argv[i]}.js`);
         scrapers.push(scraper);
     }
+} else {
+    const ls = fs
+        .readdirSync("./site-scrapers", { withFileTypes: true })
+        .filter((item) => !item.isDirectory())
+        .filter((item) => item.name !== "index.js")
+        .map((item) => item.name);
+
+    ls.map((fileName) => {
+        scraper = require(`./${fileName}`);
```

Comment:
`let scraper`

(Do we need a linter?)

Reply:
I have eslint running, and prettier. What's your question?

Reply:
I don't mean to belabor this, but `let scraper` on L6 is block scoped to L5-L10, so `scraper` on L19 is a global variable.